chat/src/main/java/net/md_5/bungee/api/Keybind.java

```diff
@@ -0,0 +1,86 @@
+package net.md_5.bungee.api;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import net.md_5.bungee.api.chat.KeybindComponent;
+
+/**
+ * All possible keybind values used by {@link KeybindComponent}.
+ */
+@RequiredArgsConstructor
+public enum Keybind
```

Comment:
Mods may have custom keybinds that they wish to use, there is no way to use that here.